### PR TITLE
Shorten method bodies

### PR DESCRIPTION
**Pull request by geigerj**
See googleapis/gax-python#104

### CODE REVIEW

**Comment by geigerj** [comment]
We should ensure compatibility with googleapis/gax-python#108

**Comment by geigerj** [comment]
cc: @garrettjonesgoogle

File: src/test/java/com/google/api/codegen/testdata/python_main_library.baseline [hunk 1]
```diff
@@ -208,24 +208,25 @@ class LibraryServiceApi(object):
...
             page_descriptors=self._PAGE_DESCRIPTORS)
-        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
-            app_name,
-            app_version,
-            self._CODE_GEN_NAME_VERSION,
-            self._GAX_VERSION,
-            platform.python_version()
-        )
-        self._headers = [('x-goog-api-client', goog_api_client)]
         self.stub = config.create_stub(
```
Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/python_main_library.baseline]: stub can be a local variable.

Comment (geigerj) [inline src/test/java/com/google/api/codegen/testdata/python_main_library.baseline]: Local, as in, private?
We want to let people drop down to the gRPC layer if needed.

Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/python_main_library.baseline]: ah, okay, I didn't realize that requirement.

**Comment by bjwatson** [comment]
Sorry @geigerj. I ran out of time to review this, due to working on the release of the logging API. I skimmed over it and nothing jumps out as being a problem, but I didn't do a proper review.

Can @garrettjonesgoogle review this instead? Thanks.

**Comment by geigerj** [comment]
No problem, I'll switch out the assignees.

**Comment by bjwatson** [comment]
Thanks, Jacob.

File: src/main/resources/com/google/api/codegen/py/main.snip [hunk 2]
```diff
@@ -178,12 +177,13 @@
...
                 @end
         @else
-            self._defaults = api_callable.construct_settings(
+            defaults = api_callable.construct_settings(
                 '{@service.getFullName}',
                 default_client_config,
                 client_config,
                 config.STATUS_CODE_NAMES,
-                timeout)
+                timeout
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/py/main.snip]: missing a comma here.

**Comment by garrettjonesgoogle** [comment]
Just one missing comma that needs to be fixed - otherwise LGTM.

File: src/test/java/com/google/api/codegen/testdata/python_main_library.baseline [hunk 3]
```diff
@@ -208,24 +208,25 @@ class LibraryServiceApi(object):
...
         default_client_config = json.loads(pkg_resources.resource_string(
             __name__, 'library_service_client_config.json'))
-        self._defaults = api_callable.construct_settings(
+        defaults = api_callable.construct_settings(
             'google.example.library.v1.LibraryService',
             default_client_config,
             client_config,
             config.STATUS_CODE_NAMES,
             timeout,
+            metadata=metadata,
```
Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/python_main_library.baseline]: As we discussed in the PR of gax-python, this is going to be `kwargs={'metadata': metadata}`

Comment (geigerj) [inline src/test/java/com/google/api/codegen/testdata/python_main_library.baseline]: Done

File: src/main/resources/com/google/api/codegen/py/main.snip (hunk 2 above)
Comment (geigerj) [inline src/main/resources/com/google/api/codegen/py/main.snip]: Done

**Comment by jmuk** [comment]
LGTM

**Pull request merged**